.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -32,6 +32,35 @@ body:
       placeholder: Steps to reproduce, e.g. "Go to [Matomo-URL] > Click on Visitors > Click Overview"
     validations:
       required: true
+  - type: input
+    id: customer-website-name
+    attributes:
+      label: Customer and Website Name
+      description: Customer Name and Website Name
+    validations:
+      required: true
```

Comment:
Based on experience from past reports most users do not want to provide that information publicly.
We can consider allowing a user to provide this data if they want, but it should not be required. This will prevent many users from reporting bugs - or they will just fill in things like `example.com`.

Reply:
Good feedback @sgiehl I will change it to not required.